meta/src/validator.rs

```diff
@@ -102,6 +102,7 @@ pub fn validate_pairs<'i>(pairs: Pairs<'i, Rule>) -> Result<Vec<&'i str>, Vec<Er
     builtins.insert("ALPHANUMERIC");
     builtins.insert("ASCII");
     builtins.insert("NEWLINE");
+    builtins.extend(::UNICODE_PROPERTY_NAMES);
```

Comment:
Are the `::` necessary here? I think `UNICODE_PROPERTY_NAMES` can simply be `use`d.

Reply:
Sorry, that's habit; I've been writing too many macros recently where you can't really import names.